TicTacToeGame.cpp

```diff
@@ -0,0 +1,97 @@
+#include <iostream>
+#include <iomanip>
+
+void initializeBoard(int a[3][3]) {
+    for(int i = 0; i < 3; i++) {
+        for(int j = 0; j < 3; j++) {
+            a[i][j] = 0;
+        }
+    }
+}
```

Comment:
A bit useless function - when you initialize board you can write int board[3][3] = {0}, it will initialize the whole matrix with zeros